### PR TITLE
Test and update the `elns` module.

### LINKED ISSUE #459: Test the `elns` module

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #459

### CODE REVIEW

**Review by unkcpz — APPROVED** [review]
Thanks @yakutovicha, I don't have any experience on eln widget, so just has a blind go-through of the changes and all seems good.

File: tests/test_elns.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    widget = ElnImportWidget()
+    assert widget.node is None
+
+
+def test_eln_export_widget(structure_data_object, mock_eln_config):
+    """Test the ElnExportWidget."""
+
+    # At the moment, we do a very minimal test of the widget, because it requires a running ELN instance.
+    # TODO: add a mock ELN instance.
```
Comment (unkcpz) [inline tests/test_elns.py:52]:
```suggestion
```
This is done by mock eln config, I assume?

File: tests/test_elns.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    assert isinstance(eln, CheminfoElnConnector)
+
+    mock_eln_config.restore()
+
+
+def test_eln_import_widget():
+    """Test the ElnImportWidget."""
+
+    # At the moment, we do a very minimal test of the widget, because it requires a running ELN instance.
+    # TODO: add a mock ELN instance.
```
Comment (unkcpz) [inline tests/test_elns.py:40]:
```suggestion
```

File: tests/test_elns.py (hunk 1 above)
Comment (yakutovicha) [reply tests/test_elns.py:52]: Not really. The config is mocked, but the sending and receiving information to/from ELN is not mocked.

**Pull request merged**